main.go

```diff
@@ -26,6 +31,11 @@ func init() {
 	boot.Add("STAT", stat.Boot, nil)
 	boot.Use("HTTP", srv.Http)
 	boot.Use("GRPC", srv.Grpc)
+
+	location.GrpcRegister()
+	tollgate.GrpcRegister()
+	crossing.GrpcRegister()
+	worker.GrpcRegister()
```

Comment:
Remove

Reply:
we agreed to migrate to init() and import package approach in another (next) PR.